test/test-433-before-001.js

```diff
@@ -0,0 +1,53 @@
+var net = require('net');
+
+var irc = require('../lib/irc');
+var test = require('tape');
+
+var MockIrcd = require('./mockircd');
+
+test('connect, register and quit', function(t) {
```

Comment:
We should name this test suite more appropriately. Perhaps `connect and nickname already in use`?